### PR TITLE
THEMES-1221: Update carousel styles

### PR DESCRIPTION
**Pull request by nschubach**
## Ticket

- [THEMES-1221](https://arcpublishing.atlassian.net/browse/THEMES-1221)

## Description

Update carousel logical properties

## Acceptance Criteria

The above logical properties are applied and tested on the gallery component.
Storybook is updated with RTL. 

## Test Steps

1. Checkout branch - `git checkout themes-1221`
2. Update dependencies - `npm i`

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed relevant documentation has been updated/added.
- [x] Add label - **ready for review** when the pull request is ready for someone to begin reviewing

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] All GitHub Actions pass
- [ ] Ran the code locally based on the test instructions.
- [ ] Checked Chromatic for Storybook changes, accepted the updates if acceptable
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.
- [ ] Approve and Add label - **ready to merge** if you are happy with the pull request
- [ ] Want another reviewer? Add the label **additional review**

[THEMES-1221]: https://arcpublishing.atlassian.net/browse/THEMES-1221?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/carousel/_index.scss [hunk 1]
```diff
@@ -4,8 +4,11 @@
 	--viewable-slides: var(--slides, 4);
 	--slide-width: var(--carousel-slide-width, 25%);
 
+	container-name: c-carousel;
+	container-type: inline-size;
 	display: grid;
 	grid-template-areas: "controls" "carousel" "indicators";
+	max-inline-size: 100cqi;
```
Comment (vgalatro) [inline src/components/carousel/_index.scss]: Are these three props (`container-name`, `container-type`, and `max-inline-size`) supposed to be here? They're also on the token, so they end up appearing twice in the final CSS.

**Review by vgalatro — CHANGES_REQUESTED** [review]
The changes look good, just noticed some duplicate properties between the `_index.scss` and the `news.json`. The changes are also missing from the `commerce.json`.

Could you add a RTL story to `index.stories.mdx` as well? So we have it whenever we update Storybook.

File: src/components/carousel/_index.scss (hunk 1 above)
Comment (nschubach) [inline src/components/carousel/_index.scss]:
```suggestion
	display: grid;
	grid-template-areas: "controls" "carousel" "indicators";
```

Comment (nschubach) [reply src/components/carousel/_index.scss]: Nope, I removed these.  I missed moving these over.

**Review by vgalatro — APPROVED** [review]
Changes look good, approved!

**Pull request merged**